src/hooks/useHint.ts

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+import sanitizeHtml from '../services/sanitize-html'
+import { RepeatableSetIndexContext } from '../form-elements/FormElementRepeatableSet'
+import useFormSubmissionModel from './useFormSubmissionModelContext'
+import { submissionService } from '@oneblink/sdk-core'
+import { localisationService } from '@oneblink/apps'
+
+const useHint = (elementHint: string | undefined) => {
+  const { formSubmissionModel, elements } = useFormSubmissionModel()
```

Comment:
this should be scoped liked the info elements

Reply:
(I'm using the above doesn't scope it)

Reply:
resolving...

Reply:
Tis.